src/plot3d.jl

```diff
@@ -58,7 +58,7 @@ for f in mplot3d_funcs
     fs = string(f)
     @eval @doc LazyHelp(axes3D,"Axes3D", $fs) function $f(args...; kws...)
         using3D() # make sure mplot3d is loaded
-        ax = gca(projection="3d")
+					PyPlot.version <= v"3.4" ? ax = gca(projection="3d") : ax = plt.subplot(projection="3d")
```

Comment:
Probably
```jl
ax = PyPlot.version <= v"3.4" ? gca(projection="3d") : plt."subplot"(projection="3d")
```

Reply:
OK, very good. (I now see how the plt."" is used.)

Reply:
Included this in the PR.